### DISCUSSION TITLE
Foreign student of OpenClassrooms online school

### DISCUSSION THREAD

**Discussion by Habibouldmoussa** [community/community#45167] [GitHub Education]
Hello, I am an Algerian student on the education platform which provides official diplomas of the French online state. I am the course of web developer, I tried to upload my study certificate, but the form of the form tells me that "the school does not correspond to your country" how can I proceed to register me to "GitHub Student" to be able to have the advantages of the pro version of Github when all my comrades have access except me?

**Comment by jge162** [comment]
Have you opened up a support ticket? They might. Be able to help you. Here is link. Click contact us. 

https://support.github.com/request/education

**Reply by Habibouldmoussa** [reply]
hello, Yes he redirects me here 
![image](https://user-images.githubusercontent.com/68819567/214277568-f60f9664-b7eb-4ec1-a831-b8c8452cfe28.png)

**Reply by jge162** [reply]
Click other support. On that link.

**Comment by Habibouldmoussa** [comment]
![image](https://user-images.githubusercontent.com/68819567/214283361-6b5814e6-0657-4127-a477-baf4265622f3.png)

you want me to repost another ticket when they closed the old one and told me to go to the forum?

**Reply by jge162** [reply]
If you did that already then no point. I did not know u tried already. Sorry.

**Reply by Habibouldmoussa** [reply]
and now what do I do?
